### DISCUSSION TITLE
How install DC from binary package in Linux Mint?

### DISCUSSION THREAD

**Discussion by vjik** [doublecmd/doublecmd#1818] [General]
I download package and run `sudo dpkg -i doublecmd-gtk_1.1.17-0+svn548~testing_amd64.deb`, but it requires dependencies:

- `doublecmd-common (= 1.1.17-0+svn548~testing)`
- `doublecmd-plugins (= 1.1.17-0+svn548~testing)`

How fix it?

My OS is Linux Mint 22 based on Ubuntu 24.04.

**Comment by JonnyTech** [comment]
https://download.opensuse.org/pub/opensuse/repositories/home:/Alexx2000/xUbuntu_24.04/amd64/doublecmd-plugins_1.1.17-0+svn548~testing_amd64.deb

https://download.opensuse.org/pub/opensuse/repositories/home:/Alexx2000/xUbuntu_24.04/all/doublecmd-common_1.1.17-0+svn548~testing_all.deb

**Reply by vjik** [reply]
Thank you!
